Common/src/main/java/com/almostreliable/unified/utils/TagMap.java

```diff
@@ -49,14 +53,17 @@ public static TagMap create(TagManager tagManager) {
         TagMap tagMap = new TagMap();
 
         for (var entry : tags.entrySet()) {
-            UnifyTag<Item> tag = UnifyTag.item(entry.getKey());
-            var holderTag = entry.getValue();
-            for (Holder<?> holder : holderTag) {
-                holder
+            UnifyTag<Item> unifyTag = UnifyTag.item(entry.getKey());
+            List<Holder<?>> tagHolders = new ArrayList<>(entry.getValue());
+
+            tagHolders.addAll(tagDelegateHelper.getHoldersForDelegate(tags, unifyTag));
```

Comment:
This still causes a generic mismatch, thus a crash on startup, because the tag manager gives you an arbitrary type of tag holders but we are only handling items.